torch/jit/_ir_utils.py

```diff
@@ -9,18 +8,20 @@ def __init__(
         self,
         insert_point_graph: torch._C.Graph,
         insert_point: Union[torch._C.Node, torch._C.Block],
-    ):
+    ) -> None:
```

Comment:
nit: Technically this one shouldn't be necessary (you only need `-> None` on __init__ if it has no parameters other than `self`).